website/docs/migration/migrating-from-mongodb.md

```diff
@@ -2,9 +2,9 @@
 sidebar_position: 2
 ---
 
-# Migrating from MongoDB
+# Migration
```

Comment:
or maybe the old name was better. It looks a bit odd now; notice how it matches the section name:

![CleanShot 2025-03-04 at 17 43 11@2x](https://github.com/user-attachments/assets/237fbf71-fa65-4bc0-bea4-8803d5841d09)